app/views/edit/build-config.html

```diff
@@ -472,7 +472,7 @@
                     <button
                       type="submit"
                       class="btn btn-primary btn-lg"
-                        ng-disabled="form.$invalid || form.$pristine || disableInputs">
+                        ng-disabled="form.$invalid || form.$pristine || disableInputs || isSourceURLInvalid()">
```

Comment:
In other words, I don't agree with the bug (unless the warning is not showing up). Added some comments in Bugzilla.